static/styles/recent_topics.scss

```diff
@@ -38,7 +38,7 @@
             position: absolute;
             top: 10px;
             right: 10px;
-            color: hsl(0, 0%, 67%);
+            color: var(--overlay-exit);
             cursor: pointer;
```

Comment:
I don't know whether we should consider this sort of thing in scope while looking at this, but should we take the opportunity to just move the duplicate copies of this to be in `static/styles/app_components.scss` as `.overlay .exit`?  I believe the implementations are all identical.  

@amanagr @SiddharthVarshney FYI.

Reply:
Ok, will do once this gets merged.